### PR TITLE
Fix formatting of see href comments

### PR DESCRIPTION
**Pull request by paul1956**
PR Feedback

## Proposed changes
- Fix a few "see href" comments were incorrectly formatted in merged PR

## Customer Impact
- Easier to read comment for developer

## Regression? 
- Maybe but only in a comment

## Risk
- None just comment change

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/12151)

### CODE REVIEW

**Review by Tanya-Solyanik — COMMENTED** [review]
I didn't finish reviewing this PR, just left some feedback.

File: src/System.Windows.Forms.Primitives/src/System/Windows/Forms/AccessibiltyExtensions.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ internal static unsafe class AccessibiltyExtensions
     /// <inheritdoc cref="PInvoke.LresultFromObject(Guid*, WPARAM, IUnknown*)"/>
     internal static LRESULT GetLRESULT(this IAccessible accessible, WPARAM wparam)
     {
-        // https://learn.microsoft.com/windows/win32/winauto/how-to-handle-wm-getobject
+        // https://learn.mcrosoft.com/windows/win32/winauto/how-to-handle-wm-getobject
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Primitives/src/System/Windows/Forms/AccessibiltyExtensions.cs:14]:
```suggestion
        // https://learn.microsoft.com/windows/win32/winauto/how-to-handle-wm-getobject
```
Bad merge?

Comment (paul1956) [reply src/System.Windows.Forms.Primitives/src/System/Windows/Forms/AccessibiltyExtensions.cs]: Possibly, its fixed now.

**Review by Tanya-Solyanik — APPROVED** [review]
Thank you!

**Pull request merged**